e2e/testsuite/testconfig.go

```diff
@@ -68,8 +68,14 @@ const (
 	// defaultConfigFileName is the default filename for the config file that can be used to configure
 	// e2e tests. See sample.config.yaml as an example for what this should look like.
 	defaultConfigFileName = ".ibc-go-e2e-config.yaml"
+
+	// defaultGenesisExportPath is the default path to which Genesis debug files will be exported to.
+	defaultGenesisExportPath = "diagnostics/genesis.json"
```

Comment:
if we end up moving the GetE2EPath to internal, maybe we can move this constant there too.